### ISSUE TITLE
Probable error in doctest doc (and python 2 in the examples)

### ISSUE THREAD

**Issue by artsiomkaltovich** [sphinx-doc/sphinx#7575]
Hello.

I am trying to configure doctest runner for my documentation. I follow the doc from here:

https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html

I am trying to following example

**To Reproduce**
```
.. testcode::

   print('Output     text.')   # brackets was added by me, example of the foc is for python 2.

.. testoutput::
   :hide:
   :options: -ELLIPSIS, +NORMALIZE_WHITESPACE

   Output text.
```

**Actual behavior**
Test failed

```
Expected:
    :hide:
    Output text.
Got:
    Output     text.
```

**Expected behavior**
Test passed

**Environment info**
- OS: Win
- Python version: [3.7.3]

```
Sphinx==2.2.1
sphinx-automodapi==0.12
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
```

**Comment by tk0miya** [comment]
It seems expectation text contains `:hide:` option. It's strange.
```
Expected:
    :hide:
    Output text.
```

On my local, it succeeded to test with your input:
```
.. testoutput::
   :hide:
   :options: -ELLIPSIS, +NORMALIZE_WHITESPACE

   Output text.
```

I guess doctest considers `:hide:` option as an output of the testcode because there are a blank line before the option. Could you check your input again, please?

**Comment by artsiomkaltovich** [comment]
Hello.

You are right, that is working fine, I've just run test with pytest, and it seems it doesn't parse rst correctly.

BTW, should the examples of print statements https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html be changed in order to add brackets to support python 3?

Regards,
Artyom.

**Comment by tk0miya** [comment]
Yes. I fixed it now on #7577.

**Comment by tk0miya** [comment]
I'm closing this now because our document was already fixed. I think there are no remaining tasks.
Thank you for reporting!
